ang/civicase/activity/services/activity-feed-measurements.service.js

```diff
@@ -17,5 +17,39 @@
 
       return topOffset;
     }
+
+    /**
+     * Returns the collective height of all contact tab children
+     *
+     * @return {Number}
+     */
+    function civicrmContactTabHeight () {
+      var height = 0;
+      var $civicrmContactTabChildren = $('.crm-contact-tabs-list').children();
+
+      _.each($civicrmContactTabChildren, function (child) {
+        height += $(child).height();
+      });
```

Comment:
why can't `$('.crm-contact-tabs-list').height()` be used? what happens if the children wrap around?

Reply:
`$('.crm-contact-tabs-list').height()` cannot be used, because `
$('.crm-contact-tabs-list').height()` gets the height of the right side panel(activity feed). Initially the height of the activity feed is very long(depends on how many elements it has), so the left tab also gets same height. So If I get that height and again set the same to activity feed, its like a circle of death and nothing happens. Thats why calculating child height is necessary.
I hope it makes sense.